lib/web_ui/lib/src/engine/skwasm/skwasm_impl/paragraph.dart

```diff
@@ -113,7 +113,7 @@ class SkwasmParagraph extends SkwasmObjectWrapper<RawParagraph> implements ui.Pa
   @override
   void layout(ui.ParagraphConstraints constraints) {
     paragraphLayout(handle, constraints.width);
-    if (!_hasCheckedForMissingCodePoints) {
+    if (!debugDisableFontFallbacks && !_hasCheckedForMissingCodePoints) {
```

Comment:
I find `if !(debugDisableFontFallbacks || _hasCheckedForMissingCodePoints)` a little bit more legible, but... it's the same.